src/Sarif.Multitool.Library/RewriteOptions.cs

```diff
@@ -10,5 +10,11 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     [Verb("rewrite", HelpText = "Enrich a SARIF file with additional data.")]
     public class RewriteOptions : SingleFileOptionsBase
     {
+        [Option(
+            's',
+            "sort-results",
```

Comment:
>results [](http://example.com/codeflow?start=18&length=7)

Why isn't this in a base class?

Reply:
Future work?

Reply:
open issue [https://github.com/microsoft/sarif-sdk/issues/2443](https://github.com/microsoft/sarif-sdk/issues/2443) for tracking